apollo-core/src/test/java/com/ctrip/framework/foundation/internals/UtilsTest.java

```diff
@@ -83,6 +83,6 @@ public void isOSWindowsFalseGivenAllUppercaseWindows() {
     @Test
     public void isOSWindowsFalseGivenAllLowercaseWindows() {
         System.setProperty("os.name", "windows 10");
-        assertFalse(Utils.isOSWindows());
+            assertFalse(Utils.isOSWindows());
```

Comment:
I think we need to also revert this change back.

Reply:
Done